### ISSUE TITLE
How to remove a group on telegram-cli?

### ISSUE THREAD

**Issue by ggrigon** [vysheng/tg#496]
Hello,

I made a new group from cli, but now i want to delete it and i can't see any option to delete the chat group.

What's the command?

**Comment by vysheng** [comment]
you can leave this group by kicking yourself from it (chat_del_user). Though you will have it in dialog list, but you will not able to send messages to it, nor you will receive new messages from it.

**Comment by luckydonald** [comment]
You might want to kick all other users first, too.

**Comment by a-sachkov** [comment]
but if sometime in the future I will be invited to a group with the same name when you try to send a message to an error

ANSWER 45
FAIL: 71: RPC_CALL_FAIL 400: PEER_ID_INVALID

because Telegram-cli is trying to send the first group of the same name

**Comment by luckydonald** [comment]
Fun thing is, with the Mac OSX native client you can leave and rejoin groups without deleting them.

**Comment by a-sachkov** [comment]
Yes, and windows and IOS native clients too. But is there any possibility to do it with telegram-cli?

**Comment by luckydonald** [comment]
I didn't manage to do that with Win...

**Comment by FabianPastor** [comment]
With repeated usernames tg-cli creates another name with \#1, \#2 and so on. Maybe it makes the same with groups. Use tab to make sure there isnt $name\#NUM chats

**Comment by a-sachkov** [comment]
Yes it does. I can use chat_info command by

chat_info chatname
chat_info chatname#1
chat_info chatname#2

and I see different chat IDs. But when I try to rename chat I get the same error:
rename_chat chatname#2 newchatname
ANSWER 45
FAIL: 71: RPC_CALL_FAIL 400: PEER_ID_INVALID
